.github/workflows/e2e.yaml

```diff
@@ -145,9 +145,6 @@ jobs:
 
     steps:
 
-      - name: Install curl on the machine
-        run: sudo apt update -y && sudo apt upgrade -y && sudo apt install -y curl
```

Comment:
Last time I checked, `curl` wasn't on the machine and the installation of `clusteradm` was failing because of that.
What has changed that we don't need it anymore?

Reply:
I think `curl` is already installed in both [Ubuntu 22.04](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#installed-apt-packages) and [Ubuntu 20.04](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#installed-apt-packages). Could it be that it was another image that you have been testing with, that doesn't include `curl`?